maximbircu/devtools-library
# Set up ios modules

Closes: #14


### What has been done
 1. Add empty iOS native framework wrapper of kotlin native generated iOS framework
 2. Add sample iOS empty app which includes native iOS wrapper framework as pod dependecy
 3. Add `SwiftLint`to both native iOS projects

### How to test
1. Open iOS native wrapper framework project and check if project builds successfully
2. Open iOS sample app project and check if it includes as dependency in `Podfile` native iOS wrapper framework. Check if sample iOS app project builds successfully

Note: For development purposes `pod` dependency of iOS framework wrapper was added to sample iOS app as local `pod` with local path. In order to add iOS framework wrapper dynamically to projects it is required to have public release in repo. We should modify framework `Podspec` and sample iOS app `Podfile` before release of first public version.
